### PR TITLE
Feeling Room ; Anxiety Animation

### LINKED ISSUE #41: Feel Room Visuals: Anxious
- [ ] The feeling room should have a dynamic background unique to the feeling.
- [ ] Animations that loop once the user starts the experience.
- [ ] A color pallet of green.

### PR DESCRIPTION
**Pull request by BrigetteDoelp**
#### Description
Adds anxiety themed animation to the the `FeelingRoom` when anxiety is selected for the mood. 
#### Motivation and Context
Part of our goals was to have an emotion specific animation for each emotion.
#### How Has This Been Tested?
- [ ] In console
- [x] In browser
- [ ] With testing files
- [ ] Other Please specify:
#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Styling / Design
- [ ] Testing
- [ ] Refactor
#### Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

#### Issues Addresses:
Resolves #41

### CODE REVIEW

File: src/Assets/forest.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...09 0 32.472 21.205 48.032 61.73-4.703 8.788-8.67 18.302-11.693 27.338-1.314 3.928.805 8.178 4.733 9.491 3.928 1.318 8.178-.805 9.492-4.733 10.682-31.932 27.159-51.697 36.299-51.697 6.195 0 17.697 9.996 28.311 31.943 10.672 22.067 17.043 49.091 17.043 72.288 0 20.725-3.849 33.474-11.767 38.975-6.213 4.316-15.379 4.96-26.087 5.065v-22.935l24.331-24.331c2.929-2.93 2.929-7.678 0-10.607-2.929-2.928-7.678-2.928-10.606 0l-13.725 13.725v-36.718c0-4.143-3.358-7.5-7.5-7.5s-7.5 3.357-7.5 7.5v80.872c-10.71-.098-19.874-.753-26.087-5.07-7.918-5.501-11.767-18.25-11.767-38.975 0-6.063.424-12.417 1.261-18.886.531-4.107-2.368-7.868-6.476-8.4-4.109-.525-7.869 2.368-8.4 6.477-.919 7.104-1.385 14.105-1.385 20.81 0 26.364 5.786 42.663 18.208 51.294.609.423 1.231.813 1.858 1.191-9.549 6.412-22.948 7.042-39.076 7.108v-78.313l11.516-11.516c2.929-2.93 2.929-7.678 0-10.607-2.929-2.928-7.678-2.928-10.606 0l-.91.91v-20.647c0-4.143-3.358-7.5-7.5-7.5s-7.5 3.357-7.5 7.5v64.157l-9.443-9.443c-2.929-2.928-7.678-2.928-10.606 0-2.929 2.93-2.929 7.678 0 10.607l18.99 18.99c.331.331.688.619 1.06.875v34.989c-16.075-.064-29.908-.717-39.461-7.352zm54.46 22.349c11.577-.048 22.736-.411 32.711-3.097v25.48h-32.711z"/></g></svg>
```
Comment (nathanielmillard) [inline src/Assets/forest.svg]: I am not sure we need these?

File: src/Assets/pause.svg [hunk 2]
```diff
@@ -1 +1 @@
-<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg" fill-rule="evenodd" clip-rule="evenodd"><path d="M10 24h-6v-24h6v24zm10 0h-6v-24h6v24zm-11-23h-4v22h4v-22zm10 0h-4v22h4v-22z"/></svg>
\ No newline at end of file
+<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg" fill-rule="evenodd" clip-rule="evenodd"><path d="M10 24h-6v-24h6v24zm10 0h-6v-24h6v24zm-11-23h-4v22h4v-22zm10 0h-4v22h4v-22z"/></svg>
```
Comment (nathanielmillard) [inline src/Assets/pause.svg:1]: Not sure the different here?

File: src/Assets/sound-bar.svg [hunk 3]
```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (nathanielmillard) [inline src/Assets/sound-bar.svg]: Also not sure about this file...

File: src/Components/FeelingRoom/FeelingRoom.js [hunk 4]
```diff
@@ -47,26 +49,50 @@ class FeelingRoom extends Component {
...
-        <div className='fineCircles' id='fineCircle4'></div>
-        <div className='fineCircles' id='fineCircle5'></div>
-        <div className='fineCircles' id='fineCircle6'></div>
-        <div className='fineCircles' id='fineCircle7'></div>
-        <div className='fineCircles' id='fineCircle8'></div>
-        <div className='fineCircles' id='fineCircle9'></div>
-        <div className='fineCircles' id='fineCircle10'></div>
-        <div className='fineCircles' id='fineCircle11'></div>
-        <div className='fineCircles' id='fineCircle12'></div>
+        <section className='hexarea'>
```
Comment (nathanielmillard) [inline src/Components/FeelingRoom/FeelingRoom.js]: Not sure if this is where you want the hexarea section?

File: src/Components/FeelingRoom/FeelingRoom.js [hunk 5]
```diff
@@ -47,26 +49,50 @@ class FeelingRoom extends Component {
...
+    anxiousAnimation('#hex7', .86, 1, 1.8, .55)
+    anxiousAnimation('#hex8', .8, .5, 2, 1.5)
+    anxiousAnimation('#hex9', .5, 1, .7, .45)
+    anxiousAnimation('#hex10', .7, 1, 2.5, 2.5)
+    anxiousAnimation('#hex11', .49, .5, .3, 2.5)
+
     return (
-      console.log('ANXIOUS')
-      //elements to animate go here
+      <section className='anxiousRoom'>
```
Comment (nathanielmillard) [inline src/Components/FeelingRoom/FeelingRoom.js:81]: Looking good, but a little long. Might have a refactor for all of these down the line.

File: src/Components/FeelingRoom/anxiousAnimations.scss [hunk 6]
```diff
@@ -0,0 +1,218 @@
+.anxiousRoom {
```
Comment (nathanielmillard) [inline src/Components/FeelingRoom/anxiousAnimations.scss:1]: This is a heafty chunk og code that I'm excited tot alk through on our next call. The end result is beautiful though :)

**Review by nathanielmillard — APPROVED** [review]
I know these hexagons caused you a lot of stress, but the end result is gorgeous.
I would probably use a hexagon svg or png in the future, but we're stronger developers from knowing how to build them with divs!
Great work.

File: src/Components/FeelingRoom/FeelingRoom.js (hunk 5 above)
Comment (Kristiannmiller) [reply src/Components/FeelingRoom/FeelingRoom.js:81]: Agreed! I'd love to see if we could refactor this to be a bit shorter

**Review by Kristiannmiller — APPROVED** [review]
This looks stunning!!! I love the hexagons and the vibe is perfect for anxious. Not too jarring, but not too soothing. Can't wait to see it with sounds

**Pull request merged**